bcgov/mds
# 3879 - Deleted Contact still remains linked to Roles

# Main

- Focus was on resource and model side, specifically in 'mine_party_appt.py'.
- Delete process works as expected only get has made the issues.

# How to test

- Go to: Mine -> Contacts: (then) delete party, refresh the page; deleted contact should not be showing again.
- Go to: Mine -> Contacts -> (choose one of the manager's parties) go to History -> deleted contact should not be showing.

# Notes

- Any notes/screenshots for this PR?
- N/A
